components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$mah-cover-basic: 676px !default;
+$fz-cover-basic-button: 16px !default;
+
+.sui-CoverBasic {
+  max-height: $mah-cover-basic;
+  position: relative;
+  width: 100%;
+
+  &.is-clickable {
+    cursor: hand;
+  }
+
+  &-gradient {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+  }
+
+  &-gradient:after {
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 25%, rgba(0, 0, 0, 0.75));
+    content: '';
+    display: inline-block;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &-image {
+    display: block;
+    height: 100%;
+    object-fit: cover;
+    width: 100%;
+  }
+
+  &-buttonList {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 0 0 52px;
+    position: absolute;
+    top: 0;
+
+    button {
+      background-color: $c-white;
+      border: 0;
+      border-radius: 4px;
+      color: $c-black;
+      font-size: $fz-cover-basic-button;
+      line-height: 1.5;
+      margin-right: 10px;
+      min-width: 90px;
+      min-height: 40px;
+      opacity: 0.75;
+      padding: 0 15px;
```

Comment:
if possible, all this declarations should use theme basic variables.
`$lh-base`
for margins, i would define `$m-small, $m-xsmall, $m-xxsmall` as @nguasch did with paddings in
https://github.com/SUI-Components/theme-basic/pull/73/files#diff-c4aed6ba387ce119c150c70302460cc6R123